assets/css/asciidoctor-mod.css

```diff
@@ -86,21 +86,6 @@ table thead tr th,table tfoot tr th,table tbody tr td,table tr td, table tfoot t
 
 .listingblock > .content { position: relative; }
 
-.listingblock:hover code[class*=" language-"]:before { text-transform: uppercase; font-size: 0.9em; color: #999; position: absolute; top: 0.375em; right: 0.375em; }
```

Comment:
Did you ever notice that hovering over a code block puts the word "Clojure" into the text and messes it up?  It's because of this. My CSS is not good enough to fix it (I suspect the intention was for the hover to appear in a less disruptive way) so I just removed it.